src/servers/trigger.ts

```diff
@@ -135,12 +118,15 @@ export async function onchainTransactionEventHandler(tx) {
 
       const price = await Prices.getCurrentPrice()
       const usdPerSat = price instanceof Error ? undefined : price
-      await NotificationsService(onchainLogger).onChainTransactionReceivedPending({
-        walletId: user.walletId,
-        amount: toSats(Number(tx.tokens)),
-        txHash: tx.id,
-        usdPerSat,
-      })
+
+      wallets.forEach((wallet) =>
+        NotificationsService(onchainLogger).onChainTransactionReceivedPending({
+          walletId: wallet.id,
+          amount: toSats(Number(tx.tokens)),
```

Comment:
I was going to say that if we're separating by wallets we might also want to sum up the relevant utxo amounts for a given wallet here instead of reporting the entire tx amount across all utxos

... but then I noticed from the type that the `lightning` library doesn't give us that level of granularity

Reply:
oh right. let me add a TODO at least.